ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -115,6 +115,65 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws-all),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-openstack
+    decorate: true
+    name: pull-ci-openshift-installer-master-e2e-openstack
+    optional: true
+    rerun_command: /test e2e-openstack
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```

Comment:
Why?

Reply:
Copied it from the aws one. Let me ask you a couple of questions:

- Should `run_if_changed` be set?
- What should it be set to?

Reply:
removed this line for now. I'd appreciate some info about how to use it properly, tho. Thanks a bunch

Reply:
You shouldn't need it